e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts

```diff
@@ -348,7 +349,8 @@ describe("scenarios > question > custom column > expression shortcuts > combine"
 
     popover().findByText("Count of rows").click();
 
-    addCustomColumn();
+    // add custom column
+    cy.findAllByTestId("action-buttons").first().icon("add_data").click();
```

Comment:
now we have 2 blocks of action buttons, so helper doesn't handle this case

Reply:
Can we create a helper for this case? I think we should create a habit of doing that whenever we introduce new UI components.

Reply:
IDK, its needed only in two places so far, also we have another helper imported here `addCustomColumn` that's why I just inlined those called to clearly show we're interested only in the first `action-buttons`

Reply:
to clarify: we do not introduce new components here, we only have cases when two groups of action buttons are rendered at the same time

Reply:
Yep not a blocker for me, I agree that it's maybe not enough of a "new component" to warrant a separate helper.